recipe/meta.yaml

```diff
@@ -26,24 +30,63 @@ requirements:
   run:
     - python
     - certifi
-    - httpcore >=1,<2
+    - httpcore >=1.0.0,<2.0.0a0
```

Comment:
you could just do `httpcore 1.*` etc

Reply:
I thought that our preference was expanding them, next time will leave them so. Thanks!